test_scripts/Polices/user_consent_of_Policies/ATF_No_user_consent_prompt_in_group.lua

```diff
@@ -0,0 +1,141 @@
+---------------------------------------------------------------------------------------------
```

Comment:
Please delete the empty spaces in section Description. 
All lines should be left aligned with no spaces as shown in template https://github.com/CustomSDL/sdl_atf_test_scripts/pull/32